train/src/config.py

```diff
@@ -20,12 +20,16 @@ def migrate(config: dict):
     if "weight_decay" not in config:
         config["weight_decay"] = 0
 
+    if "test_ratio" not in config:
+        config["test_ratio"] = 0.01
```

Comment:
あ、これ別に問題とかではないんですけど、0のが良いかも。
```suggestion
    if "test_ratio" not in config:
        config["test_ratio"] = 0
```

マイグレーションだから、以前のデフォルト値を代入しておくのが直感的な気がしていて、以前はここが0なのと同値なので。